CHANGES.txt

```diff
@@ -49,6 +49,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Ninja: Fix issue where Configure files weren't being properly processed when build run
       via ninja.
     - Added ninja mingw support and improved ninja CommandGeneratorAction support.
+    - Added user configurable setting of ninja depfile format. This helps in mingw environments
```

Comment:
please add `(via NINJA_DEPFILE_PARSE_FORMAT)` to this and the release.txt blurb

Reply:
updated in the latest